app/actors/KafkaActor.java

```diff
@@ -30,7 +31,11 @@ public KafkaActor(DataPointMapper dataPointMapper) {
                     logger.info("Received String message: {}", s);
                     ConsumerRecords<String, String> records = consumer.poll(1000);
                     List<String> jsonRecords = KafkaUtils.recordValues(records);
-                    dataPointMapper.mapDataPoints(jsonRecords);
+                    try {
+                        dataPointMapper.mapDataPoints(jsonRecords);
+                    } catch (DatapointMappingException ex) {
+                        logger.error("Caught error - discarding input: {}", ex.getMessage());
```

Comment:
This exception is only thrown by DataPointMapper#mapDataPoints, which already logs the full stack trace of the underlying cause before throwing this exception.